controllers/argocd/deployment.go

```diff
@@ -261,6 +287,14 @@ func getArgoServerCommand(cr *argoprojv1a1.ArgoCD) []string {
 	cmd = append(cmd, "--redis")
 	cmd = append(cmd, getRedisServerAddress(cr))
 
+	if useTLSForRedis {
+		cmd = append(cmd, "--redis-use-tls")
+		cmd = append(cmd, "--redis-ca-certificate", "/app/config/server/tls/redis/tls.crt")
+		if isRedisTLSVerificationDisabled(cr) {
+			cmd = append(cmd, "--redis-insecure-skip-tls-verify")
+		}
```

Comment:
Same as for the comment in `getArgoRepoCommand`, the `--redis-ca-certificate` and `--redis-insecure-skip-tls-verify` may be mutually exclusive